eng/CodeAnalysis.ruleset

```diff
@@ -486,4 +486,7 @@
     <Rule Id="IDE1007" Action="Hidden" />       <!-- UnboundIdentifier -->
     <Rule Id="IDE1008" Action="Hidden" />       <!-- UnboundConstructor -->
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.PublicApiAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.PublicApiAnalyzers" >
+    <Rule Id="RS0041" Action="Info" TESpecific="true" /> <!--Public members should not use oblivious types -->
```

Comment:
the issue is that we have nullable enable only in some files. This rule doesn't allow mixture when nullables are enabled.

Reply:
I find that very strange. Having `nullable enable` only in some files is the only usage of that directive, otherwise we would just enable it in csproj 😄